usmqe_tests/api/gluster/test_volume.py

```diff
@@ -63,10 +66,9 @@ def test_create_volume_invalid(
         valid_cluster_id,
         invalid_volume_configuration)["job_id"]
     # TODO check correctly server response or etcd job status
-    api.wait_for_job_status(
-            job_id,
-            status="failed",
-            issue="https://github.com/Tendrl/tendrl-api/issues/33")
```

Comment:
Is this invalid code style? How is this related to your change? I would rather not rewrite code of other people which still matches our code style in unrelated commits.